xla/service/gpu/BUILD

```diff
@@ -2313,6 +2313,19 @@ cc_library(
     alwayslink = True,  # Contains per-platform transfer manager registration
 )
 
+cc_library(
```

Comment:
Is there anything GPU specific in this pass? If not, can it be in xla/service directly?

Reply:
moved the files to xls/service.